.github/workflows/release-facebook.yml

```diff
@@ -0,0 +1,123 @@
+name: Release Facebook
+
+on:
+  workflow_dispatch:
+    inputs:
+      page_id:
+        description: 'Path parameter for /v19.0/{page-id}/photos endpoint'
+        required: false
+      rss_url:
+        description: 'Url of the blog RSS'
+        required: false
+      facebook_user_id:
+        description: 'User ID to access Facebook'
+        required: false
+      facebook_user_access_token:
+        description: 'User access token to access Facebook'
+        required: false
+  
+  push:
+    branches:
+      - release
+      - 317-add-workflow-for-facebook-post
```

Comment:
I noticed that the `push` (`release`) trigger won't work with this workflow since the RSS won't have been updated yet when the branch is merged.

Use `schedule` trigger. Set 1 pm (JST) everyday.

- [schedule - ワークフローをトリガーするイベント - GitHub Docs](https://docs.github.com/ja/actions/using-workflows/events-that-trigger-workflows#schedule)

Reply:
I see. I will change it to use `schedule` you suggest.

Reply:
I added the schedule trigger.